docs/source/glossary.md

```diff
@@ -18,3 +18,18 @@ Functional Programming
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
 :::::
+
```

Comment:
The definitions need to be inside the glossary directive. They should be above the five colons

Reply:
I have added the links to for the readers to refer to and added the definitions above the 5 colons. could you maybe have a look and tell me if any more changes need to be made